ivp/sim/ivpmodel.go

```diff
@@ -0,0 +1,32 @@
+package sim
```

Comment:
I don't think we need a proliferation of directories here. This can be merged with ivp/ivp.go.